deploy/buildtools/buildconfigfile.py

```diff
@@ -85,6 +95,17 @@ def __init__(self, args: argparse.Namespace) -> None:
         # retrieve the build host section
 
         build_farm_defaults_file = global_build_config_file["build_farm"]["base_recipe"]
+
+        schema_file = f"schemas/{build_farm_defaults_file}"
+        def validate_helper(val_cond: bool, error_msg: str):
```

Comment:
I'm a python typing noob, but what's the return type here? Is it pythonic to omit it if the function returns nothing? My preference would be to call that out.